scripts/e2e-tests.sh

```diff
@@ -269,6 +268,11 @@ function setup_test_cluster() {
 
   header "Setting up test cluster"
 
+  # Set the actual project the test cluster resides in
+  # It will be a project assigned by Boskos if test is running on Prow, 
+  # otherwise will be ${GCP_PROJECT} set up by user.
+  export E2E_CLUSTER_PROJECT="$(gcloud config get-value project)"
```

Comment:
make this readonly like the other vars

Reply:
Added `readonly`, and for the other two `export` vars as well.